src/thor/formlocalpath.cc

```diff
@@ -1,133 +0,0 @@
-#include <cstdint>
```

Comment:
now that we have shortcut recovery i guess this sample code is no longer need?

Reply:
Didn't know it's sample code. But tidy said there no compile commands for it and I could not see it used anywhere.

If we want to keep it as sample code, it should probably be built as part of CI since currently it seems to have a lot of errors due to bit-rot.